master/internal/experiment/bulk_action.go

```diff
@@ -128,11 +135,11 @@ func queryBulkExperiments(query *bun.SelectQuery,
 		query = query.Where("e.archived = ?", filters.Archived.Value)
 	}
 	if len(filters.States) > 0 {
-		var allStates []string
+		var selectStates []string
 		for _, state := range filters.States {
-			allStates = append(allStates, strings.TrimPrefix(state.String(), "STATE_"))
+			selectStates = append(selectStates, strings.TrimPrefix(state.String(), "STATE_"))
 		}
-		query = query.Where("e.state IN (?)", bun.In(allStates))
+		query = query.Where("e.state IN (?)", bun.In(selectStates))
```

Comment:
if you are already changing this.. why not just `states`. this var is already scoped to a 5 lines of code so it can have a short name and still be descriptive.

Reply:
✅